apps/docs/src/app/core/component-docs/icon-tab-bar/icon-tab-bar-docs.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component } from '@angular/core';
+
+import { ExampleFile } from '../../../documentation/core-helpers/code-example/example-file';
+import * as iconTabBarTextTypeHtml from '!raw-loader!./examples/icon-tab-bar-text-type-example/icon-tab-bar-text-type-example.component';
+import * as iconTabBarIconOnlyTypeHtml from '!raw-loader!./examples/icon-tab-bar-icon-only-type-example/icon-tab-bar-icon-only-type-example.component';
+import * as iconTabBarIconTypeHtml from '!raw-loader!./examples/icon-tab-bar-icon-type-example/icon-tab-bar-icon-type-example.component';
+import * as iconTabBarFilterTypeHtml from '!raw-loader!./examples/icon-tab-bar-filter-type-example/icon-tab-bar-filter-type-example.component';
+import * as iconTabBarProcessTypeHtml from '!raw-loader!./examples/icon-tab-bar-process-type-example/icon-tab-bar-process-type-example.component';
+
+@Component({
+    selector: 'fd-icon-tab-bar-docs',
+    templateUrl: './icon-tab-bar-docs.component.html',
+})
+export class IconTabBarDocsComponent {
+
+    iconTabBarTextTypeHtml: ExampleFile[] = [
+        {
+            language: 'html',
```

Comment:
Here and below add `ts` files.

![image](https://user-images.githubusercontent.com/20265336/127846817-4a7d0c1d-ce68-4ca2-a195-8a936cf9d9ee.png)